lib/reader-60-gpl.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright Roche Sequencing Solutions inc. and Daniel Alder
+
+# Description:
+#   fuzzy search for known licence names
+
+set -e
+
+package="$1"
+copyrightfile=
+if [ -f "/usr/share/doc/$package/copyright" ]; then
+  copyrightfile="/usr/share/doc/$package/copyright"
+elif [ -f "/usr/share/doc/${package%:*}/copyright" ]; then
+  copyrightfile="/usr/share/doc/${package%:*}/copyright"
+else
+  exit 0  # no copyright file found
+fi
+
+result=$(grep 'GNU General Public License' \
+    "$copyrightfile" | sed -r -e 's/[Ll]icence/License/g' | sort -u)
+if [ -n "$result" ]; then
+  echo "GPL-3"
```

Comment:
"GNU General Public License" without explicit version should be reported as GPL, not GPL-3